tests/phpunit/Integration/JSONScript/TestCases/p-0452.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Let's see what happens. I guess we could also use a "Allows value list" test in here.